src/ui/components/play-export-dialog.ts

```diff
@@ -14,53 +14,122 @@ declare global {
 @customElement('play-export-dialog')
 export class PlayExportDialog extends LitElement {
   static override styles = css`
-    .close {
-      float: right;
-    }
     dialog {
-      background-color: var(--color-interactive-background);
+      color: var(--color-neutral-content);
+      background-color: var(--color-neutral-background);
+
       border-bottom-left-radius: var(--radius);
       border-bottom-right-radius: var(--radius);
       border-top-left-radius: var(--radius);
       border-top-right-radius: var(--radius);
-      border-width: var(--border-width);
-      border-color: var(--color-neutral-border);
 
-      /* Add gap between viewport. */
-      margin-top: var(--space);
+      padding-top: var(--space);
+      padding-bottom: var(--space);
+      padding-left: var(--space);
+      padding-right: var(--space);
 
-      /* h2 has enough spacing. */
-      padding-top: 0;
+      /* RPL/Shadow/Medium */
+      box-shadow:
+        0px 4px 8px 0px var(--color-shade-10),
+        0px 6px 12px 0px var(--color-shade-25);
+
+      /* No border needed. Dialog background has sufficient contrast against the scrim. */
+      border-width: 0;
 
       /* to-do: breakpoints */
       width: 480px;
+      max-width: 90vw;
     }
-    h2 {
-      /* No spacing needed from horizontal rule. */
+
+    dialog::backdrop {
+      background-color: rgba(0, 0, 0, 0.6);
```

Comment:
CSS variable, please. I won't be able to keep these consistent otherwise :]

Reply:
Chromiume doesn't yet support css variables inside dialog::backdrop selectors.

The dialog element doesn't inherit the definitions.

Appears to have been some movement recently:
https://stackoverflow.com/a/77393321

Reply:
Thanks for the research and doc!